projects/debug-your-web-app-on-vs-code/project/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL"
+        crossorigin="anonymous"></script>
+    <script defer src="script.js"></script>
+</head>
+
+<body>
+    <div class="container w-25">
+
+        <form id="submitTodo">
+            <div class="mb-3">
+                <label for="todo-title" class="form-label">Todo Title</label>
+                <input type="text" class="form-control" id="todo-title">
+            </div>
+            <div class="mb-3">
+                <label for="numofpeople"> Number of People Assigned</label>
+                <input type="number" name="numofpeople" id="numofpeople">
```

Comment:
```suggestion
                <label for="num-of-people"> Number of People Assigned</label>
                <input type="number" name="num-of-people" id="num-of-people">
```

Suggesting this because the other uses of the `for`, name`, and `id` attributes use kebab-case for their values.